services/frontend/src/models/BlockProducer/blockProducers.js

```diff
@@ -141,7 +141,7 @@ const blockProducers = {
         const {
           data: { user_ratings: userRatings }
         } = await apolloClient.query({
-          variables: { bp, account: userAccount },
+          variables: { bp, user: userAccount },
```

Comment:
@tetogomez  

I renamed this attribute from `account` to `user` in the dabase so it matches the blockchain tables.

Also `user` is a better name for this field. 

We need to check that the frontend is updated with this change.